lib/deprecation_toolkit/rspec_hook.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "deprecation_toolkit/rspec"
+
+if defined?(RSpec)
+  RSpec.configure do |config|
+    config.add_setting(:deprecation_behavior, default: :record)
+    config.add_setting(:deprecation_path, default: 'spec/deprecations')
+
+    config.before(:suite) do
```

Comment:
Does this hook get triggered multiple time when running multiple suite?

Reply:
It'll run once at the begging. So when you run the command `bundle exec rspec` it'll run once for any test files that are setup to run. So in CI environments where the test suite is split up it should run once at the beginning.